SUMMARY.md

```diff
@@ -30,7 +30,7 @@
   * [Wasmer Rust API](https://docs.rs/wasmer-runtime/)
 * [C/C++](integrations/c/README.md)
   * [Setup your C/C++ environment](integrations/c/setup.md)
-  * [Wasmer C API](https://wasmerio.github.io/wasmer/crates/wasmer_c_api/wasm_c_api/)
+  * [Wasmer C API](https://wasmerio.github.io/wasmer/c/)
```

Comment:
It's a 404.

Reply:
yes, waiting for wasmerio/wasmer#2163 ;)